privaterelay/templates/settings.html

```diff
@@ -0,0 +1,121 @@
+{% extends "base.html" %}
+
+{% load socialaccount %}
+{% load static %}
+{% load relay_tags %}
+{% load ftl %}
+{% ftlconf bundle='privaterelay.ftl_bundles.main' %}
+
+{% block javascript %}
+  <script src="{% static 'js/settings.js' %}"></script>
+{% endblock %}
+
+
+{% block content %}
+
+{% get_social_accounts request.user as accounts %}
+<firefox-private-relay-addon-data 
+  data-fxa-subscriptions-url="{{ settings.FXA_SUBSCRIPTIONS_URL }}" 
+  data-premium-prod-id="{{ settings.PREMIUM_PROD_ID }}" 
+  data-premium-price-id="{{ settings.PREMIUM_PRICE_ID }}" 
+  data-aliases-used-val="{{ relay_addresses|length }}" 
+  data-emails-forwarded-val="{{ user_profile.emails_forwarded }}"
+  data-emails-blocked-val="{{ user_profile.emails_blocked }}"
+  {% with request.user.profile_set.first as user_profile %}
+    data-premium-subdomain-set="{{ user_profile.subdomain }}"
+  {% endwith %}
+  data-premium-enabled="{{ settings.PREMIUM_ENABLED }}"
+>
+</firefox-private-relay-addon-data>
```

Comment:
Not actually sure if this is necessary? Copied over from `profile.html`.

Reply:
76f17e7

(Btw, I'm also not sure what the `get_social_accounts` and `load` statements above do...)

Reply:
`load` is how Django templates import [custom tags and filters](https://docs.djangoproject.com/en/3.2/howto/custom-template-tags/). [`get_social_accounts` is a custom tag from `django-allauth`](https://django-allauth.readthedocs.io/en/latest/templates.html#social-account-tags) to get easier access to the user's social account objects. I don't see this template using `accounts` at all, though, so it looks like it's not needed. (And that would mean the template doesn't need to `load socialaccount` either.